### PR TITLE
Migration Guide: Standalone Application to DX Experience Cloud SPA v1

### PR DESCRIPTION
**Pull request by MichaelGoberling**
This pull request adds a migration guide that shows users how to migrate between Standalone Application and DX Experience Cloud SPA v1. 

## Description

It's easy to initialize an application as Standalone Application without realizing that the application won't be viewable in the App Builder Catalog in Adobe Experience Cloud. This guide is for users who have done the former and are looking to integrate their application with Adobe Experience Cloud. 

## Related Issue

https://github.com/adobe/aio-cli-plugin-app/issues/534

## Motivation and Context

This change shows users how to migrate between Standalone Application and DX Experience Cloud SPA v1. 

## How Has This Been Tested?

This migration guide has been tested locally with a fresh application, end to end. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+Applications built with App Builder can only be accessed via the App Builder Catalog in Adobe Experience Cloud if they have been initialized as **DX 
+Experience Cloud SPA v1**. 
+
+## How can I tell if my project is a Standalone Application?
+
+Your application will **not** have the `App Builder ExcShell` sub-heading in Adobe Exchange. 
+
+### Standalone Application 
+![image](https://user-images.githubusercontent.com/28722775/173436533-7adf7cf4-c783-41eb-8e80-d78a91c13227.png)
```
Comment (purplecabbage) [inline src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md:21]: I don't think you should use the Magento logo here.

**Review by purplecabbage — CHANGES_REQUESTED** [review]
This looks good, however I think you should use a generic/non-tm image in your screen shot.
You might want to also note that this is 1 way to do it.
It is actually also possible with only modifying config and not moving files around the file system.

File: src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md (hunk 1 above)
Comment (MichaelGoberling) [reply src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md]: Screenshot has been switched out, let me know if this one works.

File: src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        1. Navigate to Adobe Exchange 
+        2. Select **Manage** in the top toolbar 
+        3. Select **Apps Pending Review** in the top sub toolbar
+        4. Find your application in the list, click **Review**
+            1. Note: Application should have `App Builder ExcShell` sub-heading
+        5. Fill in approval notes, click **Approve** 
+    2. If not Admin
+        1. Ping Admin for approval
+        2. Wait for approval
+9. Done, your application should now be in the App Builder Catalag in Adobe Experience Cloud
```
Comment (shazron) [inline src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md:92]: spelling: `Catalag`

File: src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+3. Revoke the application (Optional: Only if application is published)
+    1. If admin
+        1. Navigate to Adobe Exchange 
+        2. Select **Manage** in the top toolbar 
+        3. Select **Approved Apps** in the top sub toolbar
+        4. Find your application in the list, click **Revoke**
+        5. Click **Revoke**
+    2. If not admin
+        1. Ping Admin to revoke
+        2. Wait for admin to revoke
```
Comment (shazron) [inline src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md:64]: The previous line capitalizes Admin, while it is lowercase here. Should be `Admin`?

File: src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+              impl: index.html
+        ```
+    6. Replace the contents of `app.config.yaml` with the following: 
+        ```
+        extensions:
+            dx/excshell/1:
+                $include: src/dx-excshell-1/ext.config.yaml
+        ```
+3. Revoke the application (Optional: Only if application is published)
+    1. If admin
```
Comment (shazron) [inline src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md:56]: Similar to next comment regarding capitalization of `Admin`

**Review by shazron — CHANGES_REQUESTED** [review]

**Review by shazron — APPROVED** [review]

**Pull request merged**